sdk/textanalytics/ai-text-analytics/samples/typescript/src/beginAnalyzeHealthcare.ts

```diff
@@ -26,7 +26,7 @@ export async function main() {
 
   const client = new TextAnalyticsClient(endpoint, new AzureKeyCredential(apiKey));
```

Comment:
do u think it is worth renaming the file to `beginAnalyzeHealthcareEntities.ts` ?